CHANGES.rst

```diff
@@ -1,3 +1,10 @@
+Version 0.1.1
+-------------
+
+Unreleased
+
+-   Replace ``posixemulation`` from Werkzeug 1.0.1 with os.rename
```

Comment:
Use code formatting for `os.rename`. Period at end of sentence. Add `:issue:` reference.